tensorflow/python/ops/special_math_ops.py

```diff
@@ -94,27 +94,40 @@ def einsum(axes, *inputs):
   """
   A generalized contraction between tensors of arbitrary dimension.
 
-  Like numpy.einsum, but does not support:
-  -- ellipses (subscripts like 'ij...,jk...->ik...')
-  -- subscripts that reduce to scalars
+  Like `numpy.einsum`, but does not support:
+  * Ellipses (subscripts like `ij...,jk...->ik...`)
+  * Subscripts that reduce to scalars
+
```

Comment:
Can you add an "Args:" section here, and after the args section, add a "Returns:" section. See other op docstrings for examples.